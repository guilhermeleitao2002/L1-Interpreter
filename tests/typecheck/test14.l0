type MoreCircle = struct { #cx:int, #cy:int, #rad:int, #color:string };
type Circle = struct { #cx:int, #cy:int, #rad:int };
type Rectangle = struct { #h:int, #w:int };
let area = fn c:Circle => {
  let pi = 3;
  println("(" + c.#cx + ", " + c.#cy + ")");
  c.#rad * c.#rad * pi
};
let c1 = { #cx=10, #cy=20, #rad=5, #color="red" };
area(c1);;