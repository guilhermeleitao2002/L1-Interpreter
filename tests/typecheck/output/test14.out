(10, 20)
75